data/load_data.js

```diff
@@ -0,0 +1,31 @@
+import { fetchGithubProfile } from "../api/github.js";
+async function load_authors(talk) {
+    let fetch_authors_promises = [];
+    talk.authors.forEach(author => {
+        fetch_authors_promises.push(fetchGithubProfile(author));
```

Comment:
I don't think that we need here any requests because all images can be found by https://github.com/{nickname}.png address